native/apache-2.0/Makefile.vc

```diff
@@ -16,38 +16,25 @@
 PROJECT = mod_jk
 # Tools
 CC = cl.exe
-LINK = link.exe
+LN = link.exe
 RC = rc.exe
-MT = mt.exe
 
 !IF !DEFINED(APACHE2_HOME) || "$(APACHE2_HOME)" == ""
 !ERROR Cannot find APACHE2_HOME. Use nmake -f Makefile.vc APACHE2_HOME=dir
 !ENDIF
 !IF !DEFINED(APACHE2_LDIR) || "$(APACHE2_LDIR)" == ""
-APACHE2_LDIR=$(APACHE2_HOME)\lib
+APACHE2_LDIR = lib
```

Comment:
This looks like it should probably continue to have the `$(APACHE2_HOME)/` prefix. Why is this being removed?